provisioner/docker/config_fedora-33.yaml

```diff
@@ -15,9 +15,9 @@
 
 docker:
         memory_limit: "4g"
-        image: "bigtop/puppet:trunk-fedora-31"
+        image: "bigtop/puppet:trunk-fedora-33"
```

Comment:
I could not pull this image yet.

Reply:
After merging this, we can run the Jenkins job that builds and pushes a new image with this PR. Then it becomes available on DockerHub.